packages/ui/src/components/input/commons/index.tsx

```diff
@@ -194,7 +193,7 @@ export const BaseInputWrapper = ({
     className,
     children,
 }: BaseInputWrapperProps): ReactElement => {
-    const [infoIcon, setInfoIcon] = useState<SVGSVGElement | null>(null);
+    const infoIconRef = useRef<HTMLDivElement | null>(null);
```

Comment:
We can't do this otherwise the popover won't update position if the anchor element is moved (yeah I know that kinda doesn't happen anyway, but [here](https://popper.js.org/react-popper/v2/#example)'s what the official popper doc has to say)

Reply:
Fixed in 89da18cc23654778f43a45577bf2968c8f3c57c0.

Reply:
Why is the `as` needed? We need to change the type of the `useState` to `HTMLDivElement` from `SVGSVGElement`.

Reply:
Oh you're right, my bad; fixed in af37f2978c4b83b36c4d1c6313db893c8f4f91ca.